src/main/scala/nl/codestar/sbtfindsecbugs/FindSecBugs.scala

```diff
@@ -36,26 +36,29 @@ object FindSecBugs extends AutoPlugin {
         val classDir = (classDirectory in Compile).value.getAbsolutePath
 
         Keys.streams.value.log.info(s"Performing FindSecurityBugs check of '$classDir'...")
-        Fork.java(
+        val result = Fork.java(
           ForkOptions(javaHome = javaHome.value, outputStrategy = Some(new LoggedOutput(streams.value.log))),
-          List("-Xmx1024m", "-cp", classpath, "edu.umd.cs.findbugs.LaunchAppropriateUI",
-            "-textui", "-html:plain.xsl", "-output", output.getAbsolutePath, "-nested:true", "-auxclasspath", auxClasspath,
-            "-low", "-effort:max", "-pluginList", pluginList, "-include", includeFile.getAbsolutePath,
-            classDir))
-      }
+          List("-Xmx1024m", "-cp", classpath, "edu.umd.cs.findbugs.LaunchAppropriateUI", "-textui",
+            "-exitcode", "-html:plain.xsl", "-output", output.getAbsolutePath, "-nested:true",
+            "-auxclasspath", auxClasspath, "-low", "-effort:max", "-pluginList", pluginList,
+            "-include", includeFile.getAbsolutePath, classDir))
```

Comment:
The change here, is that I added `"-exitcode", and then I re-arranged everthing to make it look prettier.